examples/inference/client.py

```diff
@@ -23,7 +23,7 @@ async def run_main(host: str, port: int, stream: bool = True):
         content="hello world, write me a 2 sentence poem about the moon", role="user"
     )
     cprint(f"User>{message.content}", "green")
-    iterator = client.inference.chat_completion(
+    response = client.inference.chat_completion(
```

Comment:
there is no `await` here for any of the methods? these are all synchronous I guess? are there corresponding methods where you prefix things with "a" to do async stuff?

Reply:
Yes these are all synchronous, there's corresponding aysnc methods with prefix async_: https://github.com/meta-llama/llama-stack-client-python/blob/b3b16efe13736d643e1c36cd325e2ba8d47632de/src/llama_stack_client/resources/inference/inference.py#L325